tests/testthat/helper-functions.R

```diff
@@ -374,31 +364,14 @@ tests_exit <- function() {
 #' @param port port number to _attempt_ to run server on.
 #' @param code R code to execute in a child session
 #' @return port number of the successful attempt
-run_servr <- function(directory = ".", port = 4848,
-                      code = "servr::httd(dir='%s', port=%d)") {
-  dir <- normalizePath(directory, winslash = "/", mustWork = TRUE)
-  cmd <- sprintf(
-    paste("write.table(Sys.getpid(), file='%s', append=T, row.name=F, col.names=F);", code),
-    file.path(find_test_path(), "pids.txt"), dir, port
-  )
-  system2("Rscript", c("-e", shQuote(cmd)), wait = FALSE)
+run_servr <- function(directory, port) {
```

Comment:
I don't think run_servr is used in the code anywhere anymore, is it? please delete

Reply:
sorry I only looked at R/* files, where it is not used. 
but it is used in testthat/* files